### ISSUE TITLE
Word Crates sharply increased

### ISSUE THREAD

**Issue by DicksonOrmus** [goonstation/goonstation#2500]
[LABEL] -->
[BUG]

**Describe the bug**
Until a few rounds ago number crates were the most common.  Now almost all crates are word crates, which are harder to do and way less fun.  There was no changelog on this so I thought it might be unintentional.

**Comment by F-OS** [comment]
I'm scratching my head on this. Crates have their tier, which chooses the lock type, set by 
https://github.com/goonstation/goonstation/blob/cc137f8326a3444007083aaa7dc7dc2574621c51/code/obj/storage/loot_crates.dm#L19

Which calls RarityClassRoll here,  https://github.com/goonstation/goonstation/blob/3c37ed8fc6f4dc6770d9adea472b5d8d3f6c333f/code/procs/helpers.dm#L1459 

Considering neither's been changed for 7 months and tier's not changed in any sneaky way in the doc, this may either be a coincidence or a higgs-buggson of massive proportions. I may need to try spawning 100 crates and see whether they follow the strange distribution you talked about.

**Comment by DicksonOrmus** [comment]
So I've kept an eye on it since,  and while there are still number crates... I am still noticing massively MORE word crates to the point I might only see one number crate out of eight or so some games.    Maybe the server is just rolling the opposite lottery in  consistent rare rolls on these.   I been keeping an eye on it since I posted it, determined to close it if it was just bad luck, but it's remained about the same or only slightly better.
